test-network-function/generic/suite.go

```diff
@@ -201,6 +210,25 @@ var _ = ginkgo.Describe(multusTestsKey, func() {
 	}
 })
 
+// Helper to test that the PUT didn't install new packages after starting, and report through Ginkgo.
+func testFsDiff(masterPodOc, targetPodOc *interactive.Oc) {
+	ginkgo.It(fmt.Sprintf("%s(%s) should not install new packages after starting", targetPodOc.GetPodName(), targetPodOc.GetPodContainerName()), func() {
+		targetPodOc.GetExpecter()
+		containerIDTester := containerid.NewContainerID(defaultTimeout)
+		test, _ := tnf.NewTest(targetPodOc.GetExpecter(), containerIDTester, []reel.Handler{containerIDTester}, targetPodOc.GetErrorChannel())
+		testResult, err := test.Run()
+		gomega.Expect(testResult).To(gomega.Equal(tnf.SUCCESS))
+		gomega.Expect(err).To(gomega.BeNil())
+		containerID := containerIDTester.GetID()
+
+		fsDiffTester := cnffsdiff.NewFsDiff(defaultTimeout, containerID)
+		test, _ = tnf.NewTest(masterPodOc.GetExpecter(), fsDiffTester, []reel.Handler{fsDiffTester}, masterPodOc.GetErrorChannel())
```

Comment:
Please check that the returned err is nil:
https://github.com/test-network-function/test-network-function/blob/main/pkg/tnf/test.go#L113